modelhub/modelhub/models/funnel_discovery.py

```diff
@@ -179,11 +179,9 @@ def get_navigation_paths(
         :param steps: Number of steps/locations to consider in navigation path.
         :param by: sets the column(s) to group by. If by is None or not set,
             then steps are based on the order of events based on the entire dataset.
-        :param location_stack: the location stack
-
-            - can be any slice of a :py:class:`modelhub.SeriesLocationStack` type column
-            - if None - the whole location stack is taken.
-
+        :param column: the column of which to create the paths. Can be a string of the name of the
```

Comment:
```suggestion
        :param location_stack: the column of which to create the paths. Can be a string of the name of the
```
The parameter name is `location_stack`.